ultramarine/shell-config/ultramarine-shell.sh

```diff
@@ -61,6 +61,19 @@ alias yay=_aur_helper
 alias pacaur=_aur_helper
 alias paru=_aur_helper
 
+if ! [ -x "$(command -v neofetch)" ]; then
+    neofetch() {
+        echo 'neofetch is not installed.'
+        echo 'While it is available via `sudo dnf in neofetch`, it does not come with the Ultramarine logo.'
+        echo 'We recommend fastfetch instead, which may be installed via:'
+        echo
+        echo '  sudo dnf in fastfetch'
```

Comment:
```suggestion
        echo '  sudo dnf install fastfetch'
```
(for consistency with docs)